src/Commands/StaticSiteGenerate.php

```diff
@@ -58,8 +58,9 @@ public function handle()
         if (config('statamic.editions.pro') && ! config('statamic.system.license_key')) {
             $this->error('Statamic Pro is enabled but no site license was found.');
             $this->warn('Please set a valid Statamic License Key in your .env file.');
+            $confirmationText = 'By continuing you agree that this build is for testing purposes only. Do you wish to continue?';
 
-            if (! $this->confirm('By continuing you agree that this build is for testing purposes only. Do you wish to continue?', $this->option('no-interaction'))) {
+            if (! $this->option('no-interaction') && ! $this->confirm($confirmationText)) {
```

Comment:
It seems like even if you pass `--no-interaction` in the tests, the question still gets asked.
I think maybe that's something to fix from the Laravel side.